crates/wash-lib/tests/parser/files/rust_actor_custom_build.toml

```diff
@@ -0,0 +1,9 @@
+language = "rust"
+type = "actor"
+name = "custombuildactorcomponent"
+
+[actor]
+claims = ["wasmcloud:httpserver"]
+build_artifact = "target/wasm32-wasi/release/custombuildactorcomponent.wasm"
+build_command = "cargo component build --release --target wasm32-wasi"
```

Comment:
Do we have a test case that confirms this gets parsed as expected?

Reply:
ugh FINE

Reply:
Did it 😄